packages/main/src/Select.hbs

```diff
@@ -1,15 +1,21 @@
 <div
 	class="ui5-select-root"
-	tabindex="{{tabIndex}}"
 	dir="{{dir}}"
+	tabindex="{{tabIndex}}"
+	id="{{_id}}-select"
+	role="button"
+	aria-haspopup="listbox"
+	aria-labelledby="{{_id}}-label"
+	aria-describedby="{{valueStateTextId}}"
+	?disabled="{{disabled}}"
```

Comment:
since this is an ordinary div, not an input , this should be aria-disabled, not just disabled

Reply:
Done